src/govuk/settings/_colours-organisations.scss

```diff
@@ -15,132 +15,211 @@
 /// @access public
 
 $govuk-colours-organisations: (
-  "attorney-generals-office": (
-    colour: #9f1888,
-    colour-websafe: #a03a88
+  "attorney-generals-office":
```

Comment:
Can we keep this consistent with our other maps (spacing, typography), and leave the opening parenthesis on the same line as the key?

Reply:
Mmm, not sure what happened there. Wonder if different formatting plugins (ones in project vs. ones in my code editor) got into a fight?